waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/WaggleDanceIntegrationTest.java

```diff
@@ -616,8 +611,8 @@ public void restApiGetStatus() throws Exception {
     runWaggleDance(runner);
 
     RestTemplate rest = new RestTemplateBuilder().build();
-    PrimaryMetaStore primaryMetastore = rest
-        .getForObject("http://localhost:18000/api/admin/federations/primary", PrimaryMetaStore.class);
+    PrimaryMetaStore primaryMetastore = rest.getForObject("http://localhost:18000/api/admin/federations/primary",
```

Comment:
Has anything actually changed in this class other than formatting? If not you can revert all the changes and remove it from this PR.

Reply:
Can I have an answer on the above question please? Just point out what has actually changed (if anything) or remove this class from the PR please.

Reply:
OK. I have removed it from the PR. Now one of the LOG statements looks like this:

```LOG
        .info(">>>> Partitions added: {}",
            client
                .add_partitions(Arrays
                    .asList(newPartition(hiveTable, Arrays.asList("Europe", "UK"), partitionUk),
                        newPartition(hiveTable, Arrays.asList("Asia", "China"), partitionChina))));```